### ISSUE TITLE
Show storage quota in client

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#628]
https://forums.zotero.org/discussion/46352/

Depends on zotero/dataserver#20

**Comment by aurimasv** [comment]
Just to note that, while the Sync pane is a good place to have this indicator for your personal library, I think we need some sort of other indicator in the collection tree. Ideally, I would want this to be a tooltip, but that's not possible with XUL trees. I wonder if we could start migrating to HTML-based layouts (I know it's going to be a PITA)

**Comment by dstillman** [comment]
Yes, I'd like to be able to show warning/error states for libraries. I've implemented this before in the current tree with an extra unoutlined column, but it would've meant that we'd permanently lose some horizontal space.

We badly need an HTML tree component, but since we're [not alone](https://bugzilla.mozilla.org/show_bug.cgi?id=441414), I'm hoping that someone else will write that (hopefully Mozilla, in the form of the HTML tree they're using for the debugger, which looks great but last I checked didn't provide the same sort of high performance functionality (lazy loading, etc.) that the XUL tree provides.)

**Comment by aurimasv** [comment]
Just to display a warning/error indicator, I've utilized the icon next to the label for feeds, which I think worked out well. Though displaying an error indicator is not very helpful if the user can't see what the warning/error is.

Guess we're waiting for HTML trees.
